app/routes.py

```diff
@@ -1,2 +1,245 @@
+
+    request_body = request.get_json()
+    if not valid_video_data(request_body):
+            return {"details":"Invalid data"}, 400
+
+    video.title = request_body["title"]
+    video.release_date = request_body["release_date"]
+    video.total_inventory = request_body["total_inventory"]
+    db.session.commit()
+    return jsonify(video.get_response()), 200
+
+@videos_bp.route("/<id>", methods=["DELETE"])
+def delete_customer(id):
+    video = Video.query.get(id)
+    if video == None: 
+        return {"error":f"Video ID {id} not found."}, 404
+    db.session.delete(video)
+    db.session.commit()
+    return {
+        "id":video.id
+    }, 200
+
+def valid_video_data(request_body):
+    if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+        return False
+    return True
+
+'''
+CRUD for Rental Model
+'''
+@rentals_bp.route("/check-out", methods=["POST"])
+def check_out_rental():
+    
+    request_body = request.get_json()
+
+    customer_id = request_body["customer_id"]
+    video_id = request_body["video_id"]
+
+    if customer_id == None or not isinstance(customer_id,int):
+        return {"error":"Please provide a customer ID"}, 400
+
+    if video_id == None or not isinstance(video_id,int):
+        return {"error":"Please provide a video ID"}, 400
+
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+
+    if customer == None or video == None:
+        return {"error":"Not found!"}, 404
+        
+    if video.total_inventory < 1 or (video.total_inventory - len(video.active_rentals)) < 1:
+        return {"error":"No available inventory"},400
+
+    due_date = ((datetime.today()) + (timedelta(days=7)))
+    new_rental = Rental(customer_id=customer_id, video_id=video_id, due_date=due_date)
+    db.session.add(new_rental)
+    db.session.commit()
+    return new_rental.get_rental_response(), 200
+
+
+@rentals_bp.route("/check-in", methods=["POST"])
+def check_in_rental():
+    request_body = request.get_json()
+
+    customer_id = request_body["customer_id"]
+    video_id = request_body["video_id"]
+
+    if customer_id == None or not isinstance(customer_id,int):
+        return {"error":"Please provide a customer ID"}, 400
+
+    if video_id == None or not isinstance(video_id,int):
+        return {"error":"Please provide a video ID"}, 400
+
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+    rental = Rental.query.filter_by(customer_id=customer_id,video_id=video_id).first()
+
+    if rental == None:
+        return {"error":"Rental not found"}, 400
+
+    db.session.delete(rental)
+    db.session.commit()
+
+    return {
+            "customer_id": customer_id,
+            "video_id": video_id,
+            "videos_checked_out_count": len(customer.rentals),
+            "available_inventory": (video.total_inventory - len(video.active_rentals))}
```

Comment:
You have a helper method for this in the Rental class. You could probably use it here.